mesheryctl/internal/cli/root/model/export.go

```diff
@@ -85,35 +135,38 @@ func export(modelName string, url string, output *outputDetail) error {
 	req, err := utils.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
 		utils.Log.Error(err)
-		return nil
+		return err
```

Comment:
```suggestion
		return err
```
log and return will duplicate output

Reply:
will change it everywhere